src/content/ui-api/management-api/authentication/index.md

```diff
@@ -0,0 +1,117 @@
+---
+linkTitle: Authentication
+title: Authentication for the Management API
+description: How to authenticate for the management API as a user using single sign-on, plus some information for admins regarding how to set up single sign-on with your own identity provider.
+weight: 20
+menu:
+  main:
+    identifier: uiapi-managementapi-authentication
+    parent: uiapi-managementapi
+last_review_date: 2021-04-23
+user_questions:
+  - How can I authenticate as a user for the management API?
+  - How can I inspect the ID token issued for authenticating with the management API?
+  - What requirements are there to use single sign-on with the management API?
+  - How ca I refresh my groups memberships for management API authorization?
+---
+
+# Authentication for the Management API
+
+Here we explain how you'll authenticate against the management API as a user. For admins, we provide some information regarding the requirements for the initial [single sign-on setup](#sso-requirements) and as some [technical details](#technical-details) you might want to be interested in.
+
+## Authenticating as a user {#user-auth}
+
+As a user of the management API for any given installation, you need
+
+- A **user account** in the customer's identity provider (single sign-on).
+- The management API **endpoint URL** of the installation. Alternatively, the web user interface URL.
+
+### Using kubectl gs login
+
+Assuming that you want to work with the API using `kubectl`, the easiest method to authenticate is using the [Giant Swarm plug-in]({{< relref "/ui-api/kubectl-gs/_index.md" >}}) named `gs`. It can be [installed]({{< relref "/ui-api/kubectl-gs/installation/index.md" >}}) easily using `krew` and is recommended for all users of the management API.
+
+To set up your `kubectl` configuration file, execute the following command:
+
+```nohighlight
+kubectl gs login <URL>
+```
+
+As a URL, use the management API endpoint URL (normally starting with `https://g8s.`) or the web user interface URL (normally starting with `https://happa.g8s.`) of the installation.
+
+More information can be found in the [`kubectl gs login`]({{< relref "/ui-api/kubectl-gs/login/index.md" >}}) manual page.
+
+[![Authentication flow](sso-auth-flow-with-kubectl-gs-login.svg)](sso-auth-flow-with-kubectl-gs-login.svg)
+
+In order to complete the authentication flow, `kubectl gs` will open your default web browser. If in that browser you are authenticated with your identity provider (determined e. g. via a cookie), you will see a confirmation page and can close the browser window again. However, if you weren't authenticated with your identity provider yet, you'll have to go through the authentication process you are used to. When this is done, a confirmation page will be shown.
```

Comment:
```suggestion
In order to complete the authentication flow, `kubectl gs` will open your default web browser. If in that browser you are authenticated with your identity provider (e.g: via a cookie), you will see a confirmation page and can close the browser window again. However, if you weren't authenticated with your identity provider yet, you'll have to go through the authentication process you are used to. When this is done, a confirmation page will be shown.
```

Reply:
Do you really do `e.g` without a period after the g?

Reply:
yeah scratch that, apparently i've been doing this wrong forever!